ads/aja.md

```diff
@@ -21,29 +21,20 @@ limitations under the License.
 ### Basic
 
 ```html
-  <amp-embed width="100" height="100"
+  <amp-ad width="100" height="100"
              type="aja"
              layout="responsive"
-             data-widgetIds="AMP_1,AMP_2">
-  </amp-embed>
+             data-ssp-code="ZZZZZ">
+  </amp-ad>
 ```
 
 The above code must be accompanied by AMP-enabled widgets delivered by Aja’s Account Management Team,
 do not directly install this code with existing widgets.
 
-## Parameters
 
-- widgetIds *(**mandatory**)* - Widget Id/s Provided by Account Manager.
-- htmlURL *(optional)* - The URL of the standard html version of the page.
-- ampURL *(optional)* - The URL of the AMP version of the page.
-- styleFile *(optional)* - Provide publisher an option to pass CSS file in order to inherit the design for the AMP displayed widget. **Consult with Account Manager regarding CSS options**.
+## Configuration
 
-## Troubleshooting
+Supported parameters:
 
-### Widget is cut off
-
-According to the AMP API, "resizes are honored when the resize will not adjust the content the user is currently reading.  That is, if the ad is above the viewport's contents, it'll resize. Same if it's below. If it's in the viewport, it ignores it."
-
-**Resolution**
-
- You can set an initial height of what the widget height is supposed to be. That is, instead of ```height="100"```, if the widget's final height is 600px, then set ```height="600"```. Setting the initial height ***will not*** finalize the widget height if it's different from the actual. The widget will resize to it's true dimensions after the widget leaves the viewport.
+- SSPCODE
+  - `data-aja-ssp-asi`
```

Comment:
I don't see `data-aja-ssp-asi`? Can we update the doc

Reply:
sorry, I'll fix it now!

Reply:
It was data - ssp - code, not data - aja - ssp - asi! It seems they were mistaken for CDN tags. I'm sorry